CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+cmake_minimum_required (VERSION 3.18)
+
+project(CudaBmk)
+
+set(CMAKE_CUDA_ARCHITECTURES 52 61 70 75)
+enable_language(CUDA)
+
+add_executable(cmem cmem.cu)
+add_executable(diverge diverge.cu diverge2.cu diverge3.cu path.cu)
+add_executable(global global.cu)
+add_executable(icache icache.cu icache_kernels1.cu icache_kernels2.cu icache_kernels3.cu icache_kernels4.cu)
```

Comment:
In the original `Makefile`s, this executable is built as `icache1`.

The `Makefile`s also build `icache2`, `icache3` and `icache4`.